components/filter/test/filter-tags.visual-diff.js

```diff
@@ -35,7 +35,7 @@ describe('d2l-filter-tags', () => {
 
 			before(async() => {
 				await page.$eval(selector, async(elem, width) => {
-					elem.parentNode.style.width = `${width + 140}px`; // account for label
+					elem.parentNode.style.width = `${width + 140}px`; // account for filter
```

Comment:
From what I can tell, this is leaving room for the filter, not the label - this test isn't using the component with the custom label